File: solved/Maratones competidas/Nacional 2016/gen_c.cc
#include <bits/stdc++.h>

using namespace std;

const int CASES = 100;
int main() {
  for (int i = 0; i < CASES; ++i) {
    int n = (random() % 254) + 3;
    cout << n << endl;
    for (int j = 0; j < n; ++j) {
      cout << random() % 256 + 1 << endl;
    }
  }
  return 0;
}

File: solved/Codeforces/Codeforces Round #468 (Div. 2, based on Technocup 2018 Final Round)/genc.py
from random import randint

a, b, c = randint(0, 10), randint(0, 10), randint(0, 10)

print(a + b + c)

while a > 0:
    print(1, end=' ')
    a -= 1

while b > 0:
    print(2, end=' ')
    b -= 1

while c > 0:
    print(3, end=' ')
    c -= 1

print("")
